integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/putmessage/FiksMessageStrategy.java

```diff
@@ -0,0 +1,20 @@
+package no.difi.meldingsutveksling.noarkexchange.putmessage;
+
+import no.difi.meldingsutveksling.core.EDUCore;
+import no.difi.meldingsutveksling.ks.SvarUtService;
+import no.difi.meldingsutveksling.noarkexchange.PutMessageResponseFactory;
+import no.difi.meldingsutveksling.noarkexchange.schema.PutMessageResponseType;
+
+class FiksMessageStrategy implements MessageStrategy {
+    private SvarUtService svarUtService;
+
+    FiksMessageStrategy(SvarUtService svarUtService) {
+        this.svarUtService = svarUtService;
+    }
+
+    @Override
+    public PutMessageResponseType send(EDUCore request) {
+        svarUtService.send(request); // TODO return value
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1135)